### ISSUE TITLE
false positive: RUN lines that contain brackets are wrongly linted as exec form

### ISSUE THREAD

**Issue by assaflavie** [RedCoolBeans/dockerlint#13]
For example, the following RUN line:
`RUN pip install requests[security]`
fails because it thinks it's an exec form line, probably because of the regex that looks for brackets anywhere in the line.

**Comment by TheFriendlyCoder** [comment]
I just ran into the problem as well. Another unfortunate side effect of this bug is that the linter error message is confusing at best (ie: No opening bracket found at line ### col 1).

Also worth noting is that adding escape characters to the command doesn't affect the results either (ie: RUN pip install requests\[security\] ).

**Comment by muppetjones** [comment]
Just got a related error:

```
Arguments to RUN in exec form must not contain single quotes.
```

which checks for brackets, single quotes, and `\"`. However, this line has none:

```
if [ "${x}" = "y" ];
```

If given as the first line in a `RUN` statement, it also gives this error: `No opening bracket found`.
